tests/core/test_accounts.py

```diff
@@ -51,21 +54,32 @@
 ]
 
 
-@pytest.fixture
-def PRIVATE_BYTES():
-    return b'unicorns' * 4
+PRIVATE_KEY_AS_BYTES = b'unicorns' * 4
+PRIVATE_KEY_AS_BYTES_ALT = b'rainbows' * 4
+PRIVATE_KEY_AS_OBJ = keys.PrivateKey(PRIVATE_KEY_AS_BYTES)
+PRIVATE_KEY_AS_OBJ_ALT = keys.PrivateKey(PRIVATE_KEY_AS_BYTES_ALT)
 
 
-@pytest.fixture
-def PRIVATE_BYTES_ALT(PRIVATE_BYTES):
-    return b'rainbows' * 4
+@pytest.fixture(params=[PRIVATE_KEY_AS_BYTES, PRIVATE_KEY_AS_OBJ])
```

Comment:
Extra credit: test hexstr and int keys as well.

```
In [6]: w3.toHex(b'unicorns' * 4)
Out[6]: '0x756e69636f726e73756e69636f726e73756e69636f726e73756e69636f726e73'

In [7]: w3.toHex(b'rainbows' * 4)
Out[7]: '0x7261696e626f77737261696e626f77737261696e626f77737261696e626f7773'
```

Reply:
I'll look into this tomorrow! Good call.

Reply:
I'll hold off on merging since you wanted to take a look today. Let me know if you change your mind.

Reply:
Thanks for waiting! Expanded the tests here:
https://github.com/ethereum/eth-account/pull/30/commits/335e992fa68f51d6489978620280317a3945ac3d

I ended up starting from the private key as a hexstr as the base value and creating the int, bytes, and `PrivateKey` from that. The steps I took to get the two static values for the primary and alt test values were essentially:
```
> from eth_account import Account
> acct1 = Account.create()
> address1 = acct1.address
> pkey1 = acct1.privateKey # take the hexstr from within the HexBytes(...) return value
```